tensorflow/python/data/experimental/kernel_tests/copy_to_device_test.py

```diff
@@ -35,9 +38,10 @@
 
 
 # TODO(b/117581999): add eager coverage when supported.
-class CopyToDeviceTest(test_base.DatasetTestBase):
+class CopyToDeviceTest(test_base.DatasetTestBase, parameterized.TestCase):
 
-  @test_util.deprecated_graph_mode_only
+  @combinations.generate(
+      combinations.combine(tf_api_version=[1, 2], mode=["graph"]))
```

Comment:
replace this with `test_base.graph_only_combinations()` throughout this PR

Reply:
Done